css/style.css

```
@@ -32,3 +39,6 @@ input {
   pointer-events: none;
   opacity: 0.5;
 }
+#Load-more, #Show-less{
```

please use one uniform naming convention. 

```suggestion
#load-more, #show-less{
```

---

done!